### PR TITLE
update kafka docs and defaults to v2.0.0

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

**Comment by Dieterbe** [comment]
we should also update all configs and built-in defaults to use kafka version v2.0

File: docs/kafka.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ Kafka can be used as an [ingestion option](https://github.com/grafana/metrictank
 
 # Kafka version
 
-We recommend 0.10.0.1 or higher.
+We recommend v2.0.0 or higher, as that is what we use.
```
Comment (woodsaj) [inline docs/kafka.md]:
```suggestion
Kafka v2.0.0 or newer is recommended.
```

**Comment by Dieterbe** [comment]
I have:
1) updated https://hub.docker.com/r/raintank/kafka to make latest point to v2 (commands used `docker tag raintank/kafka:v2.0.0 raintank/kafka:latest && docker push raintank/kafka:latest`). no need to update docker stack configs because they all already use "latest"
2) updated `kafka-version = 2.0` for both kafka plugins in all configs and built-in defaults
3) tested in docker-dev-custom-cfg-kafka with offset oldest and newest (and fakemetrics). behaves fine

I think this should do it. @robert-milan anything else I forgot? I thought our config file(s) had other settings related to kafka version, but apparently not.

**Comment by woodsaj** [comment]
> no need to update docker stack configs because they all already use "latest"

existing users will need to run `docker pull raintank/kafka` to have their local image updated to the new version.

**Comment by Dieterbe** [comment]
@woodsaj sure but does that mean anything should change in this PR?

**Comment by woodsaj** [comment]
Nope, just drawing attention to the fact.

**Pull request merged**